beetsplug/vocadb.py

```diff
@@ -399,10 +399,8 @@ def album_info(
                 [id for id in category.values() if id not in artists_ids]
             )
         artist_id: Optional[str] = None
-        try:
+        if artists_ids:
```

Comment:
It's a very tiny thing, but I generally go with [EAFP](https://docs.python.org/3/glossary.html#term-EAFP) rather than [LBYL](https://docs.python.org/3/glossary.html#term-LBYL) style, since from what I've heard that's what Python's devs intend people to use.

I'll still accept this change, but if you don't have strong feelings either way, then I'd prefer to stay with EAFP style.

Reply:
Didn't know about this, I'll change it back